cmd/earthly/root_cmds.go

```diff
@@ -311,6 +311,30 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name:        "web-ui",
+			Description: "Access the web UI",
+			Usage:       "Access the web UI via your default browser and print the url",
+			UsageText:   "earthly web-ui (--provider=github|--final=ci)",
+			Hidden:      true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:        "provider",
+					EnvVars:     []string{"EARTHLY_LOGIN_PROVIDER"},
+					Usage:       "The provider to use when logging into the web-ui",
+					Required:    false,
+					Destination: &app.loginProvider,
+				},
+				&cli.StringFlag{
+					Name:        "final",
+					EnvVars:     []string{"EARTHLY_LOGIN_FINAL"},
+					Usage:       "The final destination after accessing the web ui",
+					Required:    false,
+					Destination: &app.loginFinal,
+				},
```

Comment:
I'm not sure I understand the function of this flag. What are other possible values for it?

(Forgive my ignorance if this was already covered in previous conversations).

Reply:
Right now the only valid values are `information` and `auth`.
Think of it as where a user should be redirected to after login.

Reply:
I wonder if this should be a URL instead? Maybe with some validation server-side such that the URL can only be within Earthly. But I'm thinking that a URL gives us the most flexibility.

Reply:
While I like that idea (it's fairly common in login scenarios as `nextUrl`) I think that decision is outside of the scope of this PR because it already exists in our cloud backend.
If we want to add support for a url instead, I'm completely open to discussing that internally.

For this PR, I think we should either:
- Include it as-is because it's already supported.
- Decide to omit it and wait to discuss our `nextUrl` implementation.

Reply:
I think omitting it makes the most sense, if it only has one possible value and also we're planning on replacing it anyway.

Reply:
It has been removed.